File: delivery/php-sdk/models/receiver/entity/delivery/tips_entity.php
<?php
/**
 * @Author Hanqiang
 * @Date 2025/12/9
 * @PackageName: models/receiver/entity/delivery
 * @ClassName: tips_entity
 * @Description: 麦芽田配送开放平台SDK - 添加小费接口实体
 * 提供添加小费接口的请求和响应实体定义
 * @Version 1.0
 */

namespace Maiyatian\Delivery\SDK\models\receiver\entity\delivery;

/**
 * TipsRequest 添加小费请求实体
 */
class TipsRequest
{
    /**
     * 麦芽田侧运单号
     * @var string
     */
    public $source_order_no;

    /**
     * 小费金额（分）
     * @var int
     */
    public $tips;

    /**
     * TipsRequest constructor.
     *
     * @param string $sourceOrderNo 麦芽田侧运单号
     * @param int $tips 小费金额（分）
     */
    public function __construct(string $sourceOrderNo, int $tips)
    {
        $this->source_order_no = $sourceOrderNo;
        $this->tips = $tips;
    }
}

/**
 * TipsResponse 添加小费响应实体
 */
class TipsResponse
{
    /**
     * 状态码
     * @var int
     */
    public $code;

    /**
     * 响应消息
     * @var string
     */
    public $message;
}

File: delivery/go-sdk/models/receiver/entity/delivery/tips_entity.go
/**
 * @Author Hanqiang
 * @Date 2025/12/9
 * @PackageName: entity
 * @ClassName: tips_entity
 * @Description: 添加小费实体定义
 * @Version 1.0
 */

package entity

// TipsReq 添加小费请求参数
// command: tips
// 必接: 是
// 说明: 麦芽田平台调用三方配送服务为订单添加小费
type TipsReq struct {
	// SourceOrderNo 麦芽田侧运单号
	// 麦芽田平台的订单号
	SourceOrderNo string `json:"source_order_no"`

	// Tips 小费金额
	// 单位：分
	// 注意：采用累加方式，每次调用会在原有基础上增加
	Tips int64 `json:"tips"`
}
